etc/config/c++.nicole.properties

```diff
@@ -0,0 +1,76 @@
+# for all compilers
```

Comment:
Could this file get a more descriptive name? It's easy for us to understand what this file contains, but everyone else would have a harder time figuring it out

Reply:
Or, better put, is there a reason for this not to be part of the `default.win32.properties` file? (Besides the hardcoded paths)

Reply:
It's for easier startup; see `docs/WindowsNative.md`; if you have a better idea, please do tell! I just want to make sure that people can set it up easily :)

Reply:
What if the config is embed into the doc itself or as a new doc? Indeed, moving it to the docs folder and formatting it a bit with some explanation could work pretty well.

That's only my opinion so by all means, do whatever you feel works best! :)

Reply:
That'd definitely work.